src/ggrc_workflows/__init__.py

```diff
@@ -812,7 +826,7 @@ def start_recurring_cycles():
     today = date.today()
     workflows = models.Workflow.query.filter(
         models.Workflow.next_cycle_start_date <= today,
-        models.Workflow.recurrences == True  # noqa
+        models.Workflow.recurrences
```

Comment:
It would be better to rely on `sqlalchemy.true()` here:
```suggestion
        models.Workflow.recurrences == sa.true()
```